docs/release-notes/release-notes-0.16.0.md

```diff
@@ -41,10 +41,10 @@
   `openchannel`](https://github.com/lightningnetwork/lnd/pull/6956)
 
 * [`ForwardingHistory` ](https://github.com/lightningnetwork/lnd/pull/7001) now
-  enriches each forwarding event with inbound and outbound peer alias names. In
-  order for UIs to preserve the performance of this RPC the alias lookup can be 
-  skipped by specifying `skip_peer_alias_lookup`. `lncli fwdinghistory` also 
-  adds a flag `skip_peer_alias_lookup` to skip the lookup.
+  enriches each forwarding event with inbound and outbound peer alias names if
+  the new flag `PeerAliasLookup` in the `ForwardingHistoryRequest` is set to
+  true. `lncli fwdinghistory` enables this feature by default but adds a new
```

Comment:
You need to link to this PR as well for it to be picked up in the release notes check script.